cpp/src/io/avro/avro_common.h

```diff
@@ -56,6 +57,8 @@ enum type_kind_e {
   type_array,
 };
 
+using string_pair = cudf::io::detail::column_buffer::str_pair;
```

Comment:
same here, we just need to remove the pesky namespaces
```suggestion
using cudf::io::detail::column_buffer::str_pair;
```